packages/mgt-components/src/components/mgt-file-list/mgt-file-list.scss

```diff
@@ -101,4 +101,55 @@ $progress-ring-size: var(--progress-ring-size, 24px);
       font-size: $show-more-button-font-size;
     }
   }
+
+  .shared_insight_file {
+    display: flex;
+    align-items: center;
+    padding: 11px 20px;
+
+    &:hover {
+      background-color: var(--file-item-background-color, var(--neutral-layer-1));
+      cursor: pointer;
+    }
+
+    &:last-child {
+      margin-bottom: unset;
+    }
+
+    .shared_insight_file__icon {
+      width: 28px;
+      min-width: 28px;
+      height: 28px;
+      margin-right: 12px;
```

Comment:
```suggestion
      margin-end: 12px;
```

Reply:
I think you meant margin-inline-end. Applying this.